src/containers/AppContainer.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Hold and provide store to the stateless App component
+ */
+import React, { Component } from 'react';
+import {connect} from "react-redux";
+import App from "../components/App";
+
+class AppContainer extends Component{
+    render() {
+        return <App />;
+    }
+}
+function mapStateToProps(state){
```

Comment:
Good work @Karuhanga with setting up redux but shouldn't this function be written in es6 instead of es5

Reply:
Thanks for the feedback @oma0256.
I'm not sure what you mean by `es6`.
Are you perhaps speaking of using a `const` with an `anonymous function`?
In that case, I don't really see what difference that makes.

Reply:
@Karuhanga I mean arrow functions

Reply:
am with @oma0256 on this one

Reply:
Okay so this is an interesting point. I do not know if we want to impose style preferences all over the team. In my opinion, we should be flexible in this department.
I wonder what other guys and @collinewait thinks about this.

Reply:
@Karuhanga I think imposing is such a strong word to use in this case. Oma and I are simply stating our preferences. In my opinion arrow functions are much shorter and cleaner than ES5 functions.

Reply:
Hi Team, I think there is no valid reason warranting the use of ES5 when we obviously have moved on to the ES6 way of doing things... I wouldn't call it a style preference at this point.

Reply:
I totally agree with @missvicki about arrow functions being shorter and cleaner. I think, wherever possible, we should stick with the new features in ES6 that help improve our work.

Reply:
@missvicki sorry if that came off as 'strong'.
And since the consensus is this is a cleaner, better approach, then I agree. Working on that now.